File: WarehouseManagementWeb.Application/Dto/Input/Client/UpdateClientInput.cs
namespace WarehouseManagementWeb.Application.Dto.Input.Client
{
    /// <summary>
    /// Класс входной модели редактирования клиента.
    /// </summary>
    public class UpdateClientInput
    {
        /// <summary>
        /// Id клиента.
        /// </summary>
        public int Id { get; set; }

        /// <summary>
        /// Наименование клиента.
        /// </summary>
        public string? Name { get; set; }

        /// <summary>
        /// Адрес клиента.
        /// </summary>
        public string? Address { get; set; }
    }
}

File: warehouse-management-web-angular/src/app/directory/client/models/input/update-client-input.ts
/**
 * Класс входной модели редактирования клиента.
 */
export class UpdateClientInput {
  /**
   * Id клиента.
   */
  id: number = 0;

  /**
   * Наименование клиента.
   */
  name: string = '';

  /**
   * Адрес клиента.
   */
  address: string = '';
}
